perfil/person/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+from utils.infos import STATES
+
+
+class Person(models.Model):
+    civil_name = models.CharField(max_length=250)
+    cpf = models.CharField(max_length=14, unique=True)
+    gender = models.CharField(max_length=250, blank=True)
+    voter_id = models.CharField(max_length=250, blank=True)
```

Comment:
What is `voter_id`? It seems to be a kind of primary key of the "Person". And it seems to be an integer. Why do not use an integer value for this?

Reply:
Not necessarily. Sometimes a leading zero (for example, in CPF) asks us to use string field. Sometimes some unique ids have other chars than numbers. I think that **if** we're not using this field internally to relate models, string is fine – and I think this is the case since this field is not an index.